docs/cli_guide.md

```
@@ -0,0 +1,101 @@
+  --password TEXT  HiveMind password (default read from identity file)
+  --host TEXT      HiveMind host (default read from identity file)
+  --port INTEGER   HiveMind port number (default: 5678)
+  --siteid TEXT    location identifier (default read from identity file)
+  --timeout FLOAT  Seconds to wait for responsive PINGs (default: 5.0)
+  --json           Output raw JSON topology instead of ASCII tree
+  --help           Show this message and exit.
+```
+
+**Example output (ASCII tree):**
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)